lib/view/templates.py

```diff
@@ -1676,7 +1676,106 @@ def latency_projector_selector(key):
     from_source=("node_names", "histogram"),
     for_each="histogram",
     group_by=("Namespace", "Histogram"),
-    order_by="Node",
+    order_by=FieldSorter("Node"),
+)
+
+
+def stop_writes_converter_selector(edata: EntryData):
+    if "Metric" not in edata.record:
+        return None
+
+    metric = edata.record["Metric"]
+
+    if "pct" in metric:
+        return Converters.pct(edata)
+    if "bytes" in metric:
+        return Converters.byte(edata)
+    if "_ms" in metric:
+        return Converters.time_milliseconds(edata)
+
+    return Converters.scientific_units(edata)
+
+
+sw_row_yellow_format = (
+    Formatters.yellow_alert(lambda edata: edata.record["Stop-Writes"] == True),
+)
+sw_val_red_format = (
+    Formatters.red_alert(lambda edata: edata.record["Usage%"] >= 0.90),  # 90%
+)
+sw_val_yellow_format = (
+    Formatters.yellow_alert(lambda edata: edata.record["Usage%"] >= 0.75),  # 75%
+)
```

Comment:
Might not be possible but setting these based on the configured stop_writes_pct rather than hard coded would be cool.

Reply:
Remember that `stop-writes-pct` is only one of the possible stop writes triggers. This threshold is for all the triggers. The other thing to keep in mind is that these numbers are based on the configured amount. Let me give an example since it is a bit counter intuitive.  `stop-writes-pct` is the configured threshold. 
The trigger is `memory_used_bytes`/`memory-size`. So stop-writes is triggered when `stop-writes-pct` > `memory_used_bytes`/`memory-size`.  The `Usage%` is actually (`memory_used_bytes`/`memory-size`)/`stop-writes-pct`. `Usage%` tells the user how close they are to hitting stop writes.  

```
stop_writes_pct = 50
memory_used_bytes = 25
memory_size = 100

useage% = (memory_used_bytes/memory-size)/stop-writes-size = 25% / 50% = 50%
```